extensions/wikia/AdMixExperiment/scripts/AdMixExperiment.js

```diff
@@ -0,0 +1,313 @@
+require(['wikia.throttle', 'wikia.querystring'], function (throttle, querystring) {
+	$(function () {
+		var $adAndRecircWrapper;
+		var $topRightAd = $('#TOP_RIGHT_BOXAD');
+		var $topRightAdWrapper = $('#top-right-boxad-wrapper');
+		var $footer = $('#WikiaFooter');
+		var $rail = $('#WikiaRail');
+		var $recirc;
+		var $recircWrapper;
+		var $visibleElementBeforeWrapper;
+		var $window = $(window);
+
+		var adMixRecircWrapperHeight = 0;
+		var bottomMargin = 20;
+		var breakpointSmall = 1023;
+		var globalNavigationHeight = $('#globalNavigation').outerHeight(true);
+		var gapSize = 200;
+		var firstAdTopSpace = 10;
+		var noRecircAdBottomSpace = 5;
+		var recircEnabled = true;
+		var topRightAdHeight = 250;
+		var viewportHeight = $window.height();
+		var viewportWidth = $window.width();
+		var visibleElementBeforeWrapperHeight;
+
+
+		// FIXME it should be true only for ad mix variant XW-3156
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)